ado/enforce-clang-format.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
+clang-format -style=file -i stl/inc/** stl/src/** 2>&1

```

Comment:
These `**` double stars are interesting; who is parsing them? According to my understanding, this is not cmd syntax.

Reply:
Same: Billy replied to my comment that clang-format parses them.

Reply:
...judging from later changes, that must not have been the case. @BillyONeal?

Reply:
Yep!